src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public int getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
```

Comment:
You should not create a new instance of ColorSupplier inside the method. Instead, consider having a single instance of ColorSupplier as a class field and reuse it. This aligns with the checklist item that advises against creating several instances of the same class when one is sufficient.